loopd/migration.go

```diff
@@ -0,0 +1,93 @@
+package loopd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/lightninglabs/lndclient"
+	"github.com/lightninglabs/loop/loopdb"
+	"github.com/lightningnetwork/lnd/lnrpc"
+)
+
+// migrateBoltdb migrates the boltdb to sqlite.
+func migrateBoltdb(ctx context.Context, cfg *Config) error {
```

Comment:
Perhaps as a separate PR we can also add the option to migrate sqlite to postgres. Not needed now since there're no users yet on sqlite.